### ISSUE TITLE
Varargs expansion shouldn't be split

### ISSUE THREAD

**Issue by dwijnand** [olafurpg/scalafmt#763]
- **Version**: 0.5.7
- **Integration**: CLI
- **Configuration**:
```
    style = defaultWithAlign
maxColumn = 112
```

## Unexpected formatting output

Obtained, scalafmt formats code like this:
```scala
  assemblyMergeStrategy in assembly := {
    case PathList(ps @ _ *) if ps.last == "javax.inject.Named"      => MergeStrategy.first
    case PathList(ps @ _ *) if ps.last endsWith ".class"            => MergeStrategy.first
    case PathList(ps @ _ *) if ps.last endsWith "module.properties" => MergeStrategy.first
    case PathList(ps @ _ *) if ps.last == "MANIFEST.MF"             => MergeStrategy.rename
    case "LICENSE"                                                  => MergeStrategy.first
    case "NOTICE"                                                   => MergeStrategy.first
  }
```

Expected, I would like the formatted output to look like this:
```scala
  assemblyMergeStrategy in assembly := {
    case PathList(ps @ _*) if ps.last == "javax.inject.Named"      => MergeStrategy.first
    case PathList(ps @ _*) if ps.last endsWith ".class"            => MergeStrategy.first
    case PathList(ps @ _*) if ps.last endsWith "module.properties" => MergeStrategy.first
    case PathList(ps @ _*) if ps.last == "MANIFEST.MF"             => MergeStrategy.rename
    case "LICENSE"                                                 => MergeStrategy.first
    case "NOTICE"                                                  => MergeStrategy.first
  }
```

For clarity: I would like it to format `@ _*` not `@ _ *`.

**Comment by olafurpg** [comment]
Thank you for reporting! This is probably an easy fix, a good place to start would be to follow the tutorial and track where that space comes from https://github.com/olafurpg/scalafmt/blob/b52acf0d78ab7d677b6eb807b18486e9acf4111f/Tutorial.md

I suggest we keep this fix behind a configuration flag named `spaces.beforeSeqWildcard`, which will be true for now but default to false on the next 0.x./1.0 release.

Fyi,

```scala
@ q"{case PathList(ps @ _ *) => MergeStrategy.first}".structure
res2: String = """
Term.PartialFunction(Seq(Case(Pat.Extract(Term.Name("PathList"), Nil, Seq(Pat.Bind(Pat.Var.Term(Term.Name("ps")), Pat.Arg.SeqWildcard()))), None, Term.Select(Term.Name("MergeStrategy"), Term.Name("first")))))
"""
```

**Comment by olafurpg** [comment]
This got fixed by @tgodzik in 0.5.8, to use the fix

```
spaces.beforeSeqWildcard = false
```
